Cargo.toml

```
@@ -38,3 +38,4 @@ env_logger = "0.9"
 
 [features]
 check-req = []
+surplus = []
```

```suggestion
surplus = ["redis_storage"]
redis_storage = ["redis"]
```

redis dependency에는 optional 설정

레디스 쓰는 부분에는 redis_storage feature 체크